TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/feature_selection/FeatureSelectionController.java

```diff
@@ -0,0 +1,11 @@
+package com.novoda.tpbot.feature_selection;
+
+public interface FeatureSelectionController<LIST, FEATURE> {
+
+    void attachFeatureSelectionTo(LIST toAttachTo);
```

Comment:
Trying to make the feature selection menu as generic as possible so that it can be swapped out if necessary. The goal is to have most code live in core and only expose android when we need to.

Reply:
In the case of this interface, the `LIST` is the thing you want to attach each feature item to. `FEATURE` is the representation of the feature stored in the list. In android this will be the `menuItem` but could easily be an enum or anything else.